features/claims/litigator/scheme_nine/transfer_claim_draft_submit.feature

```diff
@@ -109,3 +109,85 @@ Feature: Litigator partially fills out a draft transfer claim, then later edits
     When I click View your claims
     Then I should be on the your claims page
     And Claim 'A20161234' should be listed with a status of 'Submitted' and a claimed amount of '£660.32'
+
+  @fee_calc_vcr
+  Scenario: I create a transfer claim for an Elected Case Not Proceeded
+
+    Given I am a signed in litigator
+    And My provider has supplier numbers
+    And I am on the 'Your claims' page
+    And I click 'Start a claim'
+    And I select the fee scheme 'Litigator transfer fee'
+    Then I should be on the litigator new transfer claim page
+    And I should see a page title "Enter fees for litigator transfer fees claim"
```

Comment:
(nit) Here and other places, use single quotes for consistency.

Reply:
I'd like to do this but the inconsistency is hardcoded into steps files - some steps expect single quotes (`And I am on the 'Your claims' page`) while others expect double quotes (`When I click "Continue" in the claim form`) so it's not straightforward to move to a consistent approach 😢